source/compatibility.tex

```diff
@@ -61,6 +61,24 @@
                                                         // previously threw \tcode{format_error}
 \end{codeblock}
 
+\diffref{format}
+\change
+Signature changes: \tcode{format}, \tcode{format_to}, \tcode{format_to_n},
+\tcode{formatted_size}.
+\rationale
+Enable formatting of views that are neither const-iterable nor copyable.
```

Comment:
What is "const-iterable"? the term is not defined afaikt.

Reply:
Agreed, but this is non-normative, so maybe we can get away with it.

@timsong-cpp , anything we can do about "const-iterable" editorially here?

Reply:
"only models `range` when non-const"? "can only be iterated over if they are not const"? "does not support iteration when const-qualified"? "only have non-const `begin`/`end` functions"?

Reply:
Maybe "Enable formatting of views that do not support iteration when const-qualified and are not copyable."
@burblebee , is that better?

Reply:
Sounds good to me!

Reply:
Doesn't seem like an improvement to me :disappointed: 

Is the "and are not copyable" clause a modifier to the first condition? i.e. do not support iteration when (const-qualified && !copyable) ? It's not supposed to be, but it can be read that way. "are not copyable and do not support ..." would solve that problem.

Reply:
I'm not sure how you can read it that way. Here's the current wording, for reference:

> Enable formatting of views that do not support iteration when const-qualified and are not copyable.

The second predicate "and are not copyable" has no subject, so it shares the subject with whatever comes before the "and"; the only matching subject here is "views". How would you read it to have any other subject?